bin/init.js

```diff
@@ -351,12 +443,15 @@ const getRoot = () => {
  */
 const runThemeCleanup = () => {
 	const deleteDirs = [
-		'.git',
 		'.github',
 		'bin',
 		'languages',
 	];
 
+	if ( ! isGitInitialized ) {
```

Comment:
If the user follows method 2, git initialisation will fail. Check line 142. And the cleanup process will end up removing the `.git` directory.